spec/integration/type/file_spec.rb

```diff
@@ -1594,7 +1593,6 @@ def pretty_transaction_error(transaction)
           :links => "follow")
         catalog.apply
         expect(Puppet::FileSystem.directory?(copy))
```

Comment:
This doesn't expect anything, eg if I do `expect(!Puppet::FileSystem.directory?(copy))` the test still passes. Could you change this to `expect(Puppet::FileSystem).to be_directory(copy)`?